storage/ephemeral/ephemeral.go

```diff
@@ -1,139 +1,33 @@
 package ephemeral
 
 import (
-	"errors"
-	"fmt"
 	"github.com/google/uuid"
-	"log"
-	"split-the-bill-server/storage"
 	"split-the-bill-server/types"
 	"sync"
 )
 
+// TODO: Can change types_test to pointer of types_test -> updating types_test will be displayed in all related structs but it will still not update changes in PendingGroupInvitations
 type Ephemeral struct {
-	lock            sync.Mutex
-	userStorage     map[uuid.UUID]types.User
-	nameIndex       map[string]uuid.UUID
-	passwordStorage map[uuid.UUID][]byte
-	cookieStorage   map[uuid.UUID][]types.AuthenticationCookie
+	lock      sync.Mutex
+	users     map[uuid.UUID]types.User
```

Comment:
does it make sense to also make the rest, i. e. types.User to a pointer to avoid any possible issues in the future?

Reply:
I would like to leave it as it is till the change is required. It works fine and at the moment we have no pointer to a user struct from other models which would make this change probably required.